test-cases/performance/perf-loaders-config.yaml

```diff
@@ -0,0 +1,21 @@
+# manual on creating loader AMI's: see docs/new_loader_ami.md
+aws_use_prepared_loaders: true
```

Comment:
if you want to be able to backport this PR to older branches, I would recommend adding this in separate commit

Reply:
@aleksbykov you state this should go to v14, v14 doesn't have `aws_use_prepared_loaders`